scripts/artifacts-building/containers/test-built-container-idempotency-test.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Dynamic inclusion of container
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars_files:
+    - container-vars.yml
+  vars:
+    image_name: "{{ lookup('env', 'CONTAINER_TYPE') | default('keystone', True) }}"
+  tasks:
+    - name: prepare group for integrity checking
+      add_host:
+        name: "/tmp/{{ image_name }}/"
+        groups: "lxc_container_artifact"
+        container_name: "LXC_NAME"
+        physical_host: "localhost"
+      changed_when: false
+
+- name: Check integrity
+  hosts: lxc_container_artifact
+  connection: chroot
+  # Gether facts is necessary
+  vars_files:
+    - container-vars.yml
+  vars:
+    image_name: "{{ lookup('env', 'CONTAINER_TYPE') | default('keystone', True) }}"
+  tasks:
+    - name: Fetch get-pip for comparison
+      get_url:
+        url: "{{ rpco_mirror_base_url }}/os-releases/{{ rpc_release }}/get-pip.py"
+        dest: /opt/get-pip.py
+        force: yes #Ensure it's redownloaded for testing purposes, not directly ok.
+    - name: Check apt source is pointing to rax mirror
```

Comment:
Sure we could:
- Add a task to check that files in /etc/apt contains updates or backports, registering resulting files,
- Asserting that the register doesn't hold files.

But that would be broken if we split into different repos and use the same name as upstream. Are you sure you want to do that?

Maybe we could output a list of the repos, and compare to a "known list".